results.tex

```diff
@@ -2,35 +2,32 @@ \section{Results}
 
 \subsection{Nuclear Material Inventory}
 
-\Cref{tab:sim_result} lists  \gls{EU} inventory and
```

Comment:
I thought that since this is a single simulation, the results should list the material inventory at the end of the simulation.

Now, I realize that what we care about from other EU nations is their inventory at 2050 (to send to France). I made the changes so that the table and paper lists values up to 2050 for EU nations' inventory.